lib/src/screens/receive/widgets/qr_widget.dart

```diff
@@ -188,7 +188,7 @@ class QRWidget extends StatelessWidget {
 
   Future<void> changeBrightnessForRoute(Future<void> Function() navigation) async {
     // if not mobile, just navigate
-    if (!DeviceInfo.instance.isMobile) {
+    if (!ResponsiveLayoutUtil.instance.isMobile) {
```

Comment:
this should also stay with Device Info since the used package supports only iOS and Android